setup.py

```diff
@@ -21,7 +21,7 @@
 import setuptools  # type: ignore
 
 name = "google-cloud-aiplatform"
-version = "0.3.1"
+version = "0.3.1-alpha"
```

Comment:
If this is going to be pushed out to PyPI, please use one of the schemes listed here: https://www.python.org/dev/peps/pep-0440/#examples-of-compliant-version-schemes

Reply:
We will not push this to PyPI.

I wonder if the postfix should be `experimental` to align with the current release stage naming.

Reply:
Updated to `0.4.0a1`.